content/docs/troubleshooting/logging.md

```diff
@@ -0,0 +1,43 @@
+# Centralized logging framework of Tinkerbell
+
+## Overview
+
+Tinkerbell as of today is a composition of following microservices
+- tink-server
+- hegel
+- boots
+- db
+- cli
+- nginx
+- registry
+- tink-worker
+- action containers
+
+
+Logging is crucial for any product or feature to ensure debugging and bug-free implementation. Logs of these services can be obtained using `docker logs <service name>`. For tink-worker and action containers, the steps are same and many a times it would happend that service containers has exited once the execution is completed leaving the user with an overhead of first searching the dead container `docker ps -a` to check the logs of that container. Also, logs are segregated across the nodes. So, the user will have to jump across the nodes to check the logs of the respective container or microservice. With the advent of more feature on these services microservices `docker logs` will become laborious to use from the console.
```

Comment:
`bug-free implementation`? :) nop, let's leave it out please. That's a dream!

Reply:
Let's be very clear.

Tinkerbell leverage Docker in two different ways. First, you can run the above services as a container. Second, every action is a container by itself.

Reply:
you are suggesting to segregate the explanation here in terms of docker usage for provisioner and worker?